hr_employee_calendar_planning/views/hr_employee_views.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 Tecnativa - Pedro M. Baeza
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="view_employee_form" model="ir.ui.view">
+        <field name="model">hr.employee</field>
+        <field name="inherit_id" ref="hr.view_employee_form" />
+        <field name="arch" type="xml">
+            <field name="resource_calendar_id" position="attributes">
+                <attribute name="invisible">1</attribute>
+            </field>
+            <field name="resource_calendar_id" position="after">
+                <field name="calendar_ids">
```

Comment:
@jterolc Hi! Does not seem to be displayed correctly. I tried with a label for the field but doesn't work fine.
![image](https://github.com/OCA/hr/assets/41169846/4d054a29-7332-40e8-90af-fa613769794c)

Reply:
The problem is the css rule that is applying a right and left marge to table who is not needed. I can't change this rule because is sourced from core/web...

Can you or someone help to found a clean solution?

[Captura desde 2023-06-07 13-17-48](https://github.com/OCA/hr/assets/2493980/d107e65c-88d3-4ddf-b72b-25c47a1c443b)